components/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import Link from 'next/link';
+
+export default function Footer() {
+  return (
+    <footer className="footer">
+      <div className="gel-wrap">
+        <div className="footer__social">
+          <div className="footer__social__item">
+            <a
+              href="https://www.facebook.com/roseatecards"
+              target="_blank"
```

Comment:
The w3c recommends against opening link in new tabs where possible (https://www.w3.org/TR/WCAG20-TECHS/G200.html)

Reply:
Done.